routers/users.js

```diff
@@ -12,7 +12,23 @@ router.get(path + '/', (req, res) => {
 })
 
 router.post(path + '/', (req, res) => {
```

Comment:
You should add a doc string that describes what is expected in the POST body

Reply:
Here is more of what I was thinking for the doc string, something a little more explicit
also I think the @param is only for parameters of a function, not sure how it would be used in this case though

/**
* Creates a user
* Request body format:
* {
*     username: String,
*     password: String,
*     email: String
* }
* Response body format:
* {
*     Success: Boolean,
*     message: String
* }
* Response codes:
* 201 - Successfully created
* 400 - User error
* 500 - Something went wrong
*/